frontend-gd/src/main/pages/log-in-page/login/log-in.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from 'react';
+
+import { Link } from 'react-router-dom';
+import { Redirect } from 'react-router-dom';
+import FBlogin from '../fb-login';
+import GoogleLog from '../google-login';
```

Comment:
Importing as GoogleLog, but it's being exported as GoogleLogin?

Reply:
Unsure but I think since it's the default export then it doesn't matter. But, obviously for consistency's sake we should make em match. Honestly the auth stuff is a mess I'm glad you're gonna take a look at it.